Dockerfile

```diff
@@ -0,0 +1,63 @@
+# Use the base Python 3.6 image.  It is sufficient for accomplishing
+# most of the stuff, and includes some handy build tools like gcc.
+#
+# NOTE This could be changed to something like `python:latest` if we
+# find that there is no dependency on 3.6 itself.
+FROM python:3.6
```

Comment:
There's no dep on 3.6. It currently runs on 3.5 or 3.6, and should be able to run on any stable python >= 3.5 (if it doesn't, that's a bug IMO).